infer/src/dotnet/ResourceLeakCSDomain.ml

```diff
@@ -118,18 +118,27 @@ let assign lhs_access_path rhs_access_path held =
     | exception Caml.Not_found ->
         ()
   in
+  let compare_expr_base ((base1, _)) ((base2, _)) = 
+    if AccessPath.equal_base base1 base2 then true
+    else false
+  in
+  let construct_access_path ((base, _)) ((_, accesses)) = (base, accesses) in
   let one_binding access_path count held =
     match
       AccessPath.replace_prefix ~prefix:rhs_access_path ~replace_with:access_path lhs_access_path
     with
-    | Some base_access_path ->
+    | Some base_access_path -> 
         add_type_map access_path base_access_path ;
-        ResourcesHeld.add base_access_path count held
-    | None ->
+        ResourcesHeld.add base_access_path count held 
+    | None -> 
```

Comment:
Minor: These spurious whitespace changes make this harder to review, especially since GitHub does a poor job of displaying them. Can you try installing ocamlformat with `make devsetup` and apply it to future PRs with `make fmt`?

Reply:
I have tried as you suggested. However, I got error and `make fmt` failed.
![image](https://user-images.githubusercontent.com/63893533/154319296-66c57532-3124-4b55-9f27-2a62f3b55ea6.png)
Do you know how to fix this?

Reply:
Right, that's this issue, sorry about that:
https://github.com/facebook/infer/blob/84f6b5f25acb574676f81616113b611587580ee0/infer/src/.ocamlformat#L8-L26

Alternatively you can `opam pin ocamlformat.0.19.0-24-g2596a5c7 -k git https://github.com/danielmercier/ocamlformat.git#master --no-action && opam install ocamlformat odoc-parser.0.9.0` but subsequent `make devsetup` will undo it. A new release of dune is coming that will finally get rid of this issue.

Reply:
Fixed. Thanks!